.travis.yml

```diff
@@ -0,0 +1,12 @@
+language: php
+
+before_script:
+  - wget https://getcomposer.org/composer.phar
+  - chmod +x composer.phar
+  - ./composer.phar update nothing
+  - ./composer.phar install
```

Comment:
If you "update" no need for install. Please use only install, since it will use the "lock" file instead of updating dependencies.